scripts/gen/tpcc.sh

```diff
@@ -24,6 +27,28 @@ Usage: $0 [-f] [-w] [-s server] [pgurl,...]
   exit 1
 }
 
+function is_special_type() {
+  special_type=1
+  current_result_good=1
+  case $1 in
```

Comment:
Can't we specify per benchmark args for these machines?
You know the json config could have bench args section and each section can have e.g. "tpcc", which allows to add more arguments for those machines?
```
 "benchArgs": {
      "io": "-s"
    }
```
You won't need to hardcode anything .

Reply:
Thanks for your suggestion. Create https://cockroachlabs.atlassian.net/browse/CR-86 to track this, let me introduce this change post test round 1, don't want to introduce more major changes at this point as am wrapping up tpcc test round 1.